homework.md

```diff
@@ -4,110 +4,111 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+CSS customizes the appearance and the layout of the webpage.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language to make webpages interactive.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1. keep track of the history of the changes. it is simply used for version control.
+2. allow multiple people do their own changes and merge them into one source.
+3. allow you revert to specific versions should you need to.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy the project and make changes without affecting the original project.
 
-## Question 10 
+## Question 10
 
-What happens when you clone a repostory?
+What happens when you clone a repository?
 
-It send it to a friend
+You copy an existing repository to your local computer.
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+it notifies the author of the repository that you have completed some tasks.
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+let you see the previous versions and changes.
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+You send the changes of your local repository to Github.
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+It is a web-based hosting service for software development and version control using Git.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Save the changes to the local version of repo.
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual Studio Code
 
 ## Question 17
 
 What do you use "Commit Messages" for?
 
-To record what changes you made to the code
+To make clear and helpful messages adding what changed and why.
 
 ## Question 18
 
 Where can I look to find help with Git?
 
-The Git Cheatsheet!
+It can be found on https://git-scm.com/doc
```

Comment:
This looks great! One addition that you can make in your answer to this question is to include the following link of GitHub documentation: https://git-scm.com/doc . It is a lifesaver when you want an overall view and detailed explanation of solving issues. Otherwise - Well done!

Reply:
Is there any other place where we can find help with Git? For instance, most of the git commands we will need have been explained in the class. Do we have any useful page in our CYF material which contains useful instructions for Git? :)